requirements/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
issue: Requirements need to be generated with Python 3.8, we should ideally have a CI check that validates this.